script/build.js

```diff
@@ -26,10 +25,11 @@ const sourceDir = '../content/pages';
 
 const smith = Metalsmith(__dirname);
 
+// TODO(crew): Change port back before merge to master.
 const optionDefinitions = [
   { name: 'buildtype', type: String, defaultValue: 'development' },
   { name: 'no-sanity-check-node-env', type: Boolean, defaultValue: false },
-  { name: 'port', type: Number, defaultValue: 3000 },
+  { name: 'port', type: Number, defaultValue: 4000 },
```

Comment:
I need to add this to the README.md. I changed this so I could run the API and `vets-website` at the same time on my machine.